hiroaki-core/build.gradle

```diff
@@ -10,14 +10,17 @@ dependencies {
     // linters
     ktlint "com.github.shyiko:ktlint:0.31.0"
 
+    // Junit
+    implementation 'junit:junit:4.13'
```

Comment:
Added this explicitly since we depend on it quite a bit. This was coming transitively from `okhttp.mockwebserver` before.